Breadcrumb docs (#474)
telerik/blazor-docs: components/breadcrumb/events.md

---
title: Events
page_title: Breadcrumb Events
description: Events of the Breadcrumb for Blazor.
slug: breadcrumb-events
tags: telerik,blazor,breadcrumb,events
published: True
position: 30
---

# Breadcrumb Events

This article explains the events available in the Telerik Breadcrumb for Blazor:

* [OnItemClick](#onitemclick)

## OnItemClick

The `OnItemClick` event fires when the user clicks or taps on a Breadcrumb item and before any navigation occurs. It receives object of type `BreadcrumbItemClickEventArgs` which exposes the following fields:

* `Item` - an object you can cast to your model class to obtain the clicked data item.
* `IsCancelled` - specifies whether the event is canceled and the built-in action is prevented.

> **The event will not fire for the last item or if the Breadcrumb Item has been disabled.**

Reviewer: ```suggestion
**The event will not fire for the last item and for disabled items.**
```

> You can use the `OnItemClick` event to react to user choices in a Breadcrumb without using navigation to load new content automatically.

Reviewer: ```suggestion
You can use the `OnItemClick` event to react to user choices in a Breadcrumb, and load new content without using navigation.
```
